src/test/java/spoon/test/method/MethodTest.java

```diff
@@ -218,4 +220,16 @@ public void test_addFormalCtTypeParameterAt_throwsOutOfBoundsException_whenPosit
 		assertThrows(IndexOutOfBoundsException.class,
 				() -> method.addFormalCtTypeParameterAt(1, typeParam));
 	}
+
+	@Test
+	public void testGetAllMethodsInnerClassExtended() {
+		// contract: implicit static nested interfaces are correct handled in getAllExecutables and dont throw an error.
```

Comment:
Depending on whether you want the test to verify that there's no crash, or that the correct amount of executables are returned, I would update the contract to state that.

Like I mentioned, invoking a method in a test implicitly tests that it does not crash, and so if you test anything in addition to the fact that the method does not crash (e.g. that it returns the expected amount of executables), the contract does not need to state that it tests that the method doesn't crash. That was a messy sentence but I hope you understand what I mean.

Reply:
It was a crash while parsing the JDK. I wanted to include a real assertion to guarantee that it's not only not crashing, but working correctly. But you are correct, "Does not crash" is not a useful assertion if we can check if the result is correct.